### DISCUSSION TITLE
Using neon-bindings node modules in api functions fails with  cannot open shared object file: No such file or directory

### DISCUSSION THREAD

**Discussion by jp-burford** [vercel/community#5677] [Help]
### Summary

I have created an app and I am testing including native node modules in the api.  The node modules are created using neon.  The app runs fine locally and builds in vercel.  When I run the serverless function I receive an error from nextjs-node-loader : vercel/path0/apps/jokeioke/.next/server/chunks/index.node: cannot open shared object file: No such file or directory.

### Example

https://jokeioke-2y0k1x7wm-outcrop-solutions.vercel.app/api/journal_entry

### Steps to Reproduce

Here is how I am referencing the dependency in my app: 
![image](https://github.com/vercel/community/assets/48831852/4cdd8bb0-3fcf-44f6-a912-028791038470)

Here is the actual function: 
![image](https://github.com/vercel/community/assets/48831852/f370dd91-56f4-4daa-9c9a-a45a8b4a25cf)

This is my next config: 
![image](https://github.com/vercel/community/assets/48831852/32a31cb5-6280-4dda-9b59-f36ffd3436e2)

This is the package.json from my bindings crate: 
![image](https://github.com/vercel/community/assets/48831852/f4695c81-5914-42d0-8386-b43ae697152a)

Here is a link to the repo: https://github.com/outcrop-solutions/monorepo

**Comment by sagoez** [comment]
Are there any updates on this one? @jp-burford where you able to solve it?
